src/app/components/schedule/schedule.component.ts

```diff
@@ -0,0 +1,73 @@
+import { Component, OnInit } from '@angular/core';
+import { Trip } from 'src/app/models/trip';
+import {ScheduleService} from 'src/app/services/schedule-service/schedule.service';
+import { AuthService } from 'src/app/services/auth-service/auth.service';
+
+
+@Component({
+  selector: 'app-schedule',
+  templateUrl: '../trips.component.html',
+})
+export class ScheduleComponent implements OnInit 
+{
+  public caption='Trip Schedule';
+  public trips: Trip[];
+  public riderTrips: Trip[]=[];
+  public id: number;
+  
+  constructor(private serv: ScheduleService, private auth: AuthService) 
+  { 
```

Comment:
For consistency across the codebase, please move opening curly brackets to the same line as the method declaration.